packages/flutter/lib/src/painting/alignment.dart

```diff
@@ -88,22 +87,22 @@ abstract class AlignmentGeometry {
   /// into a concrete [Alignment] using [resolve].
   ///
   /// {@macro dart.ui.shadow.lerp}
-  static AlignmentGeometry lerp(AlignmentGeometry a, AlignmentGeometry b, double t) {
+  static AlignmentGeometry? lerp(AlignmentGeometry? a, AlignmentGeometry? b, double t) {
     assert(t != null);
```

Comment:
no longer necessary right?

Reply:
We have to wait for #61042 to be fixed before removing null asserts on parameters.